projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.ts

```diff
@@ -46,17 +52,35 @@ export class SearchBoxComponent {
    * for example when we click inside the search result section.
    */
   private ignoreCloseEvent = false;
+  chosenWord = '';
+  public subscription: Subscription;
 
   /**
    * The component data is optional, so that this component
    * can be reused without CMS integration.
    */
 
+  constructor(
+    searchBoxComponentService: SearchBoxComponentService,
+    componentData: CmsComponentData<CmsSearchBoxComponent>,
+    winRef: WindowRef,
+    // tslint:disable-next-line: unified-signatures
+    routingService: RoutingService
+  );
+  /**
+   * @deprecated
```

Comment:
As mentioned in the DM, it would be best to add comments, such as `projects/storefrontlib/src/cms-components/navigation/search-box/search-box-component.service.ts`